includes/modules/analytics/google/class-api.php

```diff
@@ -54,38 +54,39 @@ private function setup() {
 
 		$tokens      = Authentication::tokens();
 		$this->token = $tokens['access_token'];
-		$this->refresh_token_on_login();
 	}
 
 	/**
 	 * Refresh access token when user login.
 	 */
-	public function refresh_token_on_login() {
+	public function refresh_token() {
```

Comment:
Please update the function reference here as well.

https://github.com/rankmath/seo-by-rank-math/blob/master/rank-math.php#L396

Reply:
Thanks, we have fixed it internally.